tests/network/probes.go

```diff
@@ -376,7 +376,7 @@ func pointIpv6ProbeToSupportPod(pod *corev1.Pod, probe *v1.Probe) (*v1.Probe, er
 }
 
 func withMasqueradeNetworkingAndFurtherUserConfig(opts ...libvmi.Option) []libvmi.Option {
-	return append(libnet.WithMasqueradeNetworking(), opts...)
+	return append([]libvmi.Option{libnet.WithMasqueradeNetworking()}, opts...)
```

Comment:
I think the reason `withMasqueradeNetworkingAndFurtherUserConfig` existed is gone now and you can drop this helper.

Reply:
What do you mean? It's still used in a bunch of places (e.g. [here](https://github.com/kubevirt/kubevirt/blob/dcc0fcb5096d5bb232ee70d3c113c8d300552487/tests/network/probes.go#L134)).
TBH I'm not sure what's the reason it existed in the first place. Can we leave this to a follow-up?

Reply:
Yes, it is used in several places and now it can be removed and the options accumulated directly without this helper.

I won't insist, but IMO it is a fit in this PR.

Reply:
Removed the function :) Thanks!